custodian/vasp/handlers.py

```diff
@@ -1013,8 +1013,11 @@ def check(self):
         """
         try:
             v = Vasprun(self.output_filename)
-            # check whether bandgap is zero and tetrahedron smearing was used
-            if v.eigenvalue_band_properties[0] == 0 and v.incar.get("ISMEAR", 1) < 0:
+            # check whether bandgap is zero, tetrahedron smearing was used
+            # and relaxation is performed.
+            if (v.eigenvalue_band_properties[0] == 0 and
+                v.incar.get("ISMEAR", 1) < -3 and 
+                v.incar.get("NSW", 0) <= 1):
```

Comment:
Looking at this, I realize that the logic is actually inverted. That's why the existing smearing test is failing. This should be `and v.incar.get("NSW", 0) > 1:` That way the handler will activate `check=True` for relaxations and not activate for statics.

Reply:
@rkingsbury, I am pretty emberrest by this mistake. Thanks a lot for catching this and also thanks for explaining how to do unit tests and linting correctly for this project.

I am guessing since the tests now pass with the updated logic and the correct pycodestyle, that I do not have to write a new `IncorrectSmearingHandlerStaticTest`? The INCAR in `test_files/scan_metal` has `NSW = 99` and `ISMEAR = -5`, and thus correctly triggers the handler I think.

Of course I am happy to follow your instructions for the new test if it is needed, even if the old one now completes OK.

Reply:
Thanks @MichaelWolloch; no problem. You're correct that after changing the logic the existing test now passes, and you've resolved all the linting errors which is good. 

It would still be preferable if you could write an additional unit test to confirm that `IncorrectSmearingHandler` is not triggered during static calculations. Would you mind giving that a try? I know writing tests is tedious, but it's critical to keeping our codes from breaking as they evolve.

Reply:
I will certainly do that. Hopefully I will find time today, otherwise you can expect something tomorrow! I am glad to be able to help a little, since I am profiting a lot from the work you all are doing within the materialsproject.